### PR TITLE
George - handle login

### PR DESCRIPTION
**Pull request by georgezsiga**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -41,4 +41,6 @@ dependencies {
     compile 'com.google.code.gson:gson:2.7'
 
     testCompile 'com.squareup.okhttp3:mockwebserver:3.7.0'
+    compile 'javax.inject:javax.inject:1'
```
Comment (adamgyulavari) [inline app/build.gradle]: @georgezsiga this is not needed

File: app/build.gradle [hunk 2]
```diff
@@ -41,4 +41,6 @@ dependencies {
...
+    compile 'org.apache.ant:ant:1.8.0'
```
Comment (adamgyulavari) [inline app/build.gradle]: @georgezsiga this neither

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/ServiceFactory.java [hunk 3]
```diff
@@ -9,15 +9,14 @@
 
 public class ServiceFactory {
 
-
+private static final String SERVER_URL = "https://tribes-of-lagopus.herokuapp.com/";
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/ServiceFactory.java]: @georgezsiga if you have endpoints like `"/login"` you dont need the `/` at the end of the url

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (georgezsiga) [inline app/build.gradle]: done

**Comment by georgezsiga** [comment]
done

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**